### ISSUE TITLE
about  result

### ISSUE THREAD

**Issue by T-USST** [aRI0U/RandLA-Net-pytorch#53]
=== EPOCH 50/50 ===
Training loss: 2.6826021	Validation loss: 2.5952491	
Accuracy      |     0 |     1 |     2 |     3 |     4 |     5 |     6 |     7 |     8 |     9 |    10 |    11 |    12 |    13 |    OA
Training:     | 0.519 | 0.139 | 0.128 | 0.360 | 0.269 | 0.003 | 0.003 | 0.142 | 0.234 | 0.124 | 0.017 | 0.002 | 0.017 | 0.006 | 0.273
Validation:   | 0.767 | 0.027 | 0.043 | 0.116 | 0.139 | 0.000 | 0.000 | 0.152 | 0.199 | 0.075 | 0.000 | 0.000 | 0.010 | 0.000 | 0.197
IoU           |     0 |     1 |     2 |     3 |     4 |     5 |     6 |     7 |     8 |     9 |    10 |    11 |    12 |    13 |  mIoU
Training:     | 0.369 | 0.058 | 0.044 | 0.259 | 0.170 | 0.000 | 0.000 | 0.032 | 0.055 | 0.026 | 0.002 | 0.000 | 0.002 | 0.001 | 0.073
Validation:   | 0.281 | 0.021 | 0.019 | 0.088 | 0.081 | 0.000 | 0.000 | 0.028 | 0.039 | 0.025 | 0.000 | 0.000 | 0.003 | 0.000 | 0.054

Why was my training so bad? What went wrong

**Comment by wangzihao77** [comment]
haoxiongdi,niyetaicanle,huangemoxingpaoquba...
